llvm/lib/Target/AArch64/AArch64SelectionDAGInfo.cpp

```diff
@@ -32,16 +32,41 @@ AArch64SelectionDAGInfo::AArch64SelectionDAGInfo()
 
 void AArch64SelectionDAGInfo::verifyTargetNode(const SelectionDAG &DAG,
                                                const SDNode *N) const {
+  switch (N->getOpcode()) {
+  case AArch64ISD::ADC:
```

Comment:
These might be fixed by #140112

Reply:
Did you mean #150328? I wonder what made @AZero13 create this PR after commenting on #150313 that does the same and a little more.

Reply:
> Did you mean https://github.com/llvm/llvm-project/pull/150328?

I did. I don't know how I managed to paste the completely wrong number.